packages/nimble-components/src/icon-base/index.ts

```diff
@@ -20,8 +25,33 @@ export class Icon extends FoundationElement {
     public constructor(/** @internal */ public readonly icon: NimbleIcon) {
         super();
     }
+
+    public override connectedCallback(): void {
+        super.connectedCallback();
+        this.forwardAriaLabelToSvg();
+    }
+
+    private ariaLabelChanged(): void {
+        this.forwardAriaLabelToSvg();
+    }
+
+    private forwardAriaLabelToSvg(): void {
+        const svg = this.shadowRoot?.querySelector('svg');
+        if (!svg) {
+            return;
+        }
+        if (this.ariaLabel) {
```

Comment:
Unresolving because the request for a test case was not addressed.

Reply:
Good call! The test actually caught an issue. In `if (this.ariaLabel ?? false) {`, when the label is an empty string, the condition becomes the empty string, just like it would without the `?? false`, so it ends up removing the attribute. Had to just do explicit `this.ariaLabel !== null`.

Reply:
Ah yea I goofed. Then you probably need to check for undefined as well.